module-api/src/main/java/com/yapp/sharefood/auth/controller/AuthController.java

```diff
@@ -59,4 +71,13 @@
 
         return ResponseEntity.created(userCreateUri).build();
     }
+
+    @DeleteMapping("/api/v1/auth/deletion")
+    @ApiResponses({
+            @ApiResponse(code = 200, message = "[success] 회원 탈퇴 성공")
+    })
+    public ResponseEntity<Void> withdraw(@ApiIgnore @AuthUser User user) {
+        return ResponseEntity.ok().build();
+    }
```

Comment:
restapi 에서 path에 id 값도 넣어주세요
```suggestion
    @DeleteMapping("/api/v1/auth/{userId}/deletion")
    @ApiResponses({
            @ApiResponse(code = 200, message = "[success] 회원 탈퇴 성공")
    })
    public ResponseEntity<Void> withdraw(@ApiIgnore @AuthUser User user, @Pahtvariable("userId") Long userId) {
        return ResponseEntity.ok().build();
    }
```

Reply:
그리고 token 정보 path variable 정보가 일치 하지 않는 경우 권한을 제어 하는 것도 필요할 거 같습니다.
response code는 forbidden입니다.